manifests/init.pp

```diff
@@ -281,6 +282,7 @@
   Optional[String] $rabbitmqadmin_package                                                          = $rabbitmq::params::rabbitmqadmin_package,
   Array $archive_options                                                                           = $rabbitmq::params::archive_options,
   Array $loopback_users                                                                            = $rabbitmq::params::loopback_users,
+  Boolean $service_restart                                                                         = $rabbitmq::params::service_restart,
```

Comment:
I can change it to whichever you like. I went with service_restart because when I first checked if this feature existed I greped for the word 'restart'. Also the kafka module uses service_restart as its variable name.

Reply:
I can see it both ways, I don't have a strong opinion.

Reply:
Anyway, can you squash down one more time? I lean a bit towards restart as well, even though notify is closer to what Puppet is doing under the hood.

Reply:
suqashed